libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/service/SessionCredentialsService.java

```diff
@@ -119,4 +168,15 @@ public void deleteSessionCredentialsForCri(String ipvSessionId, String criId)
                     HTTPResponse.SC_SERVER_ERROR, ErrorResponse.FAILED_TO_DELETE_CREDENTIAL);
         }
     }
+
+    private static boolean isNameOnlyJourney(CoiSubjourneyTypes coiSubjourneyTypes) {
+        return coiSubjourneyTypes != null
+                && (coiSubjourneyTypes.getPath().equals(GIVEN_NAMES_ONLY.getPath())
+                        || coiSubjourneyTypes.getPath().equals(FAMILY_NAME_ONLY.getPath()));
```

Comment:
Again, could extract the list to a constant

```suggestion
        return List.of(GIVEN_NAMES_ONLY, FAMILY_NAME_ONLY).contains(coiSubjourneyTypes);
```